src/com/facebook/buck/jvm/java/PrebuiltJar.java

```diff
@@ -155,6 +155,11 @@ public boolean getRequiredForSourceOnlyAbi() {
     return javadocUrl;
   }
 
+  @Override
+  public boolean IsDesugarEnabled() {
+    return true;
```

Comment:
Does this need to be done for aars as well?

Reply:
May be but I have no examples to back this up with. I saw both examples of aar files as java 8 classes and already predexed. My guess is that it won't hurt to dex them one more time in case they are pure java 8 compiled but again I a have no test for that or real use case.